src/lib/tests/get-link.test.ts

```diff
@@ -0,0 +1,60 @@
+// To run tests, execute `npm run test -- src/lib/tests/translate-link.test.ts` from the root directory
+
```

Comment:
```suggestion
```
I suggest just removing this part.

Reply:
Removed! https://github.com/Automattic/studio/pull/1375/commits/08f3cb29a78961829290e59c3d56160fe0428afe